### PR TITLE
Enable implicit inexact object lint

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: src/VariableSizeGrid.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ type ItemMetadata = {|
   offset: number,
   size: number,
 |};
-type ItemMetadataMap = { [index: number]: ItemMetadata };
+type ItemMetadataMap = { [index: number]: ItemMetadata, ... };
```
Comment (bvaughn) [inline src/VariableSizeGrid.js]: I'm unfamiliar with this "..." syntax, and my attempts to find it in the docs are failing. Pointers?

Comment (bvaughn) [reply src/VariableSizeGrid.js]: Looks like CI is unfamiliar with it also 😆 

Screen Shot 2019-06-11 at 8 27 11 AM

Comment (TrySound) [reply src/VariableSizeGrid.js]: Explicit inexact object. Part of transition to exact by default.
https://github.com/facebook/flow/releases/tag/v0.101.0

**Comment by TrySound** [comment]
Looks like it's not yet supported by babel-eslint

File: src/VariableSizeGrid.js (hunk 1 above)
Comment (TrySound) [reply src/VariableSizeGrid.js]: Will add it later then.

Comment (bvaughn) [reply src/VariableSizeGrid.js]: Gotcha. Thanks. I don't think they've published the 101 notes internally yet. I've only read v100.

**Pull request closed without merging**